dashboard/app/models/pd/workshop.rb

```diff
@@ -376,4 +378,14 @@ def effective_num_hours
   def professional_learning_partner
     ProfessionalLearningPartner.find_by_contact_id organizer.id
   end
+
+  # @return [Boolean] true if a Code Studio account and section membership is required for attendance, otherwise false.
+  def account_required_for_attendance?
+    case course
```

Comment:
Good call. It's the inverse though: `![Pd::Workshop::COURSE_COUNSELOR, Pd::Workshop::COURSE_ADMIN].include?(course)`